### ISSUE TITLE
Pretrained distiller model behaves differently on each run

### ISSUE THREAD

**Issue by kfmn** [s3prl/s3prl#384]
I am trying to extract features using pretrained distillHuBERT model provided by S3PRL. My code follows the guidelines from distiller/README:

```
import torch
from s3prl.hub import distilhubert
import librosa
import sys

def process_chunk(sig, model):
    wavs = [torch.from_numpy(sig)]
    results = model(wavs)
    # The representation used in the paper
    representation = results["paper"]
    return representation

def process_wav_file(filename, model):
    chunk_size = 1000000
    extra_right_context = 10000
    sig, sr = librosa.load(filename, sr=None)
    return process_chunk(sig, model)

pretrained_model = distilhubert()

file=sys.argv[1]
results = process_wav_file(file, pretrained_model)
print(results.shape, results)
```

But each time I run this code, I observe different values of extracted features (although input signal is always the same). 
Even If I run code once and compute features several times (repeat last two lines) they are different from time to time

It looks very strange and unexpected. Please explain me the reasons and the way to obtain reproducible results.

**Comment by kfmn** [comment]
I've found a solution. After model loading in 
```
pretrained_model = distilhubert()
```
it should be switched into evaluation mode by
```
pretrained_model.eval()
```
